### ISSUE TITLE
Reconsider "readability" requirement for conceptual colors

### ISSUE THREAD

**Issue by flowchartsman** [flowchartsman/clarion#1]
By conforming to WCAG-AA, the conceptual colors are readable against theme backgrounds, but this might not be strictly necessary, since many of the places they might appear will be in decorations or badges, and these should stand out more from the black foreground color. It might be a good idea to have the concept colors represent the base color and then derive a WCAG-compliant readable version.

**Comment by flowchartsman** [comment]
Latest branch defines three WCAG contrast levels, one of which is "UI", representing the "non-text" ratio of 3:1. Moving forward, Clarion will try and use non-text styling as much as possible for important information, opting for background colors, and UI elements such as badges and gutter blocks wherever possible. Fingers crossed for https://github.com/microsoft/vscode/issues/143470
